Python-path-traversal-files-main/CJuanvip/CMEOptions/sm.py

```
@@ -0,0 +1,173 @@
+
+	script_dir = os.path.dirname(os.path.abspath(__file__))
+	dest_dir = os.path.join(script_dir, "options")
+	if not os.path.exists(dest_dir):
+		os.mkdir(dest_dir)
+
+	file_name = "OOI_{0}".format(settlement_date.strftime("%m_%d_%Y"))
+
+	with open("{0}".format(os.path.join(script_dir, "{0}.tex".format(file_name))), "w") as f:
+		f.write(tex)
+
+	subprocess.call("pdflatex {0}".format(os.path.join(script_dir, "{0}.tex".format(file_name))), shell=True)
```

Ruff (0.13.3)

`subprocess` call with `shell=True` identified, security issue

(S602)